StRoot/StEvent/StTriggerData2022.cxx

```diff
@@ -144,15 +144,17 @@ void StTriggerData2022::readData(const TriggerDataBlk2022* data, int bs) {
     memset(mQT4,0,sizeof(mQT4));
     memset(mEQ1,0,sizeof(mEQ1));
     memset(mEQ2,0,sizeof(mEQ2));
+    memset(mEQ4,0,sizeof(mEQ4));
     memset(mxq,0,sizeof(mxq)); memset(tmxq,0,sizeof(tmxq));
     memset(eq3,0,sizeof(eq3)); memset(teq3,0,sizeof(teq3));
     memset(bbq,0,sizeof(bbq)); memset(tbbq,0,sizeof(tbbq));
-    memset(qt1,0,sizeof(qt1)); memset(tqt1,0,sizeof(tqt1));
-    memset(qt2,0,sizeof(qt2)); memset(tqt2,0,sizeof(tqt2));
-    memset(qt3,0,sizeof(qt3)); memset(tqt3,0,sizeof(tqt3));
-    memset(qt4,0,sizeof(qt4)); memset(tqt4,0,sizeof(tqt4));
+    //memset(qt1,0,sizeof(qt1)); memset(tqt1,0,sizeof(tqt1));
```

Comment:
Shouldn't we zero those fields anyway? Ideally, we'd remove the fields that are not present.